includes/Experiments.php

```diff
@@ -400,6 +400,18 @@ public function get_experiments(): array {
 				'description' => __( 'Enable inserting media element from external link', 'web-stories' ),
 				'group'       => 'editor',
 			],
+
+			/**
+			 * Author: @brookegraham
+			 * Issue: #8832
+			 * Creation date: 2021-08-30
+			 */
+			[
+				'name'        => 'enableTaxonomySupport',
+				'label'       => __( 'Taxonomy', 'web-stories' ),
```

Comment:
Nit: I'd use plural form here.

```suggestion
				'name'        => 'enableTaxonomiesSupport',
				'label'       => __( 'Taxonomies', 'web-stories' ),
```